src/commands/http-get.php

```diff
@@ -11,6 +12,7 @@
 class HttpGet implements Command {
   private $curl;
   private $url;
+  private $seconds = 120;
```

Comment:
$seconds could be anything, how about renaming it to $timeout so its clear what the value is used for.